### DISCUSSION TITLE
Is there a way to annotate an infra-as-code line to ignore errors?

### DISCUSSION THREAD

**Discussion by creffett** [aquasecurity/trivy#2859] [Q&A] [answered]
Is there a way to annotate a line in infrastructure-as-code (specifically CloudFormation) so that Trivy ignores a particular error on that line? cfsec and tfsec had this feature (e.g. https://aquasecurity.github.io/tfsec/v0.61.3/getting-started/configuration/ignores/) but I can't seem to find documentation or discussion about a similar feature for Trivy.

**Comment by amaldonado-hcg** [comment]
it is not documented on trivy but it works using the ID in this form:

```
resource "aws_security_group_rule" "my-rule" {
    type = "ingress"
    cidr_blocks = ["0.0.0.0/0"] //tfsec:ignore:AVD-AWS-0104
}
```

**Reply by ilyakruchinin** [reply]
This doesn't work for me (well, using Yaml at least with #tfsec:ignore:AVD-AWS-0104)

**Comment by nikpivkin** [comment, marked as answer]
Trivy supports ignore by inline comments for Terraform and CloudFormation. https://aquasecurity.github.io/trivy/v0.53/docs/scanner/misconfiguration/#skipping-resources-by-inline-comments
